### ISSUE TITLE
Something about Superresolution part

### ISSUE THREAD

**Issue by pouletdatrid** [TristaZeng/ZS-DeconvNet#1]
Does FiJi plugin do super-resolution? The only result I get from using the software is  denoising.

**Comment by TristaZeng** [comment]
Yes it does. Take _Plugins > ZS-DeconvNet > predict_ for example, the default setting is to show the super-resolution result, but you can tick the "show denoising result", and the super-resolution and denoising result will both show.
![default_predict_setting](https://github.com/TristaZeng/ZS-DeconvNet/assets/64056496/55bbb73e-f22a-475a-83a2-c242a868425b)
As you can see, the window on the left (with twice the input size) is the super-resolution result, and the window on the right is the denoising result.
![sr_and_den_results](https://github.com/TristaZeng/ZS-DeconvNet/assets/64056496/d5964b43-ed62-44ad-9389-5122a4dce00e)

**Comment by pouletdatrid** [comment]
Thanks for your prompt reply. I seem to have chosen the wrong zip. Can I rebuild a 9 frame sim? I used ZS-DeconvNet-2D-SIM-CPPs.zip and then I put in a 9 frame sim, but I still ended up with twice the size of the 9 frame sim.

**Comment by pouletdatrid** [comment]
> Yes it does. Take _Plugins > ZS-DeconvNet > predict_ for example, the default setting is to show the super-resolution result, but you can tick the "show denoising result", and the super-resolution and denoising result will both show. ![default_predict_setting](https://user-images.githubusercontent.com/64056496/271463305-55bbb73e-f22a-475a-83a2-c242a868425b.png) As you can see, the window on the left (with twice the input size) is the super-resolution result, and the window on the right is the denoising result. ![sr_and_den_results](https://user-images.githubusercontent.com/64056496/271463299-d5964b43-ed62-44ad-9389-5122a4dce00e.png)

Thanks for your prompt reply. I seem to have chosen the wrong zip. Can I rebuild a 9 frame sim? I used ZS-DeconvNet-2D-SIM-CPPs.zip and then I put in a 9 frame sim, but I still ended up with twice the size of the 9 frame sim.

**Comment by TristaZeng** [comment]
About that, sorry you cannot put in 9 raw SIM frames. ZS-DeconvNet-SIM deals with reconstructed SIM images, i.e., we further improve the already super-resolved SIM images. You would have to put in one reconstructed SIM image.
![R2R_SIM](https://github.com/TristaZeng/ZS-DeconvNet/assets/64056496/1949a4c2-4e87-4f56-a5c7-b7622e22a2d4)
As you can see in this diagram, the actual input of ZS-DeconvNet-SIM is the reconstructed SIM image. But the re-corruption procedure should be done on the raw SIM images, if you want to perform training.
